css/global.css

```diff
@@ -0,0 +1,433 @@
+}
+.nav__hamburger::after {
+  top: 0.8rem;
+}
+
+.nav__hamburger:hover,
+.nav__hamburger:hover::before,
+.nav__hamburger:hover::after {
+  background-color: var(--color-secondary);
+}
+
+.nav__icons {
+  flex-shrink: 0;
+  justify-content: space-between;
+  color: var(--color-white);
+  padding-top: 4px; /* hack due to icon specs */
+}
+
+.nav__icons > * {
+  font-size: 2.5rem;
+  cursor: pointer;
+  transition: color 0.3s;
+}
+
+.nav__icons > *:hover {
+  color: var(--color-secondary);
+  font-weight: 500;
+}
+
+/*
+ * MAIN
+ */
+main {
+  background-color: var(--color-white);
+  flex: 1 0 auto; /* For footer placement */
+}
+
+/*
+ * FOOTER
+ */
+footer {
+  display: grid;
+  grid-template-columns: auto 12rem 30rem;
+  padding: var(--padding-narrow);
+  
+  background-color: var(--color-primary);
+  background-image: linear-gradient(var(--color-primary), var(--color-dark));
+  color: var(--color-white);
+}
+
+footer > * {
+  display: flex;
+  flex-direction: column;
+  padding: var(--padding-wide)
+}
+
+.footer__nav {
+  display: flex;
+}
+
+.footer__nav > a {
+  color: var(--color-white);
+  font-size: var(--font-size);
+  margin-bottom: var(--margin-narrow);
+  transition: 0.3s;
+}
+
+.footer__nav > a:hover {
+  color: var(--color-secondary);
+}
+
+.social_media {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+}
+
+.social_media > p {
+  margin-bottom: var(--margin-narrow);
+  align-self: flex-start;
+}
+
+.social_media > div > a {
+  color: var(--color-white);
+  font-size: var(--heading-5);
+  margin-left: 0.2rem;
+  transition: 0.3s;
+}
+
+.social_media > div > a:hover {
+  color: var(--color-secondary);
+}
+
+.newsletter {
+  border-left: solid 2px var(--color-white);
+  display: flex;
+  justify-content: center;
+}
+
+.newsletter > *:not(:last-child) {
+  margin-bottom: var(--margin-narrow);
+}
+
+#newsletter_form {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+#newsletter_form > * {
+  margin: var(--margin-narrow) 0;
+}
+
+#email {
+  flex: 1 0 auto;
+  margin-right: var(--margin-narrow);
+  padding: var(--padding-narrow) var(--padding-wide);
+  border-radius: 2rem;
+}
+
+#newsletter_form > button {
+  border-radius: 2rem;
+}
+
+/* Tablets and ipads */
+@media (max-width: 52rem) { 
+  .logo > span {
+    display: none;
+  }
+
+  .nav__hamburger {
+    display: block;
+    z-index: 2;
+  }
+
+  .nav__menu {
+    order: 2;
+    margin-left: var(--margin-mid);
+  }
+
+  .nav__menu > ul {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    right: -15rem;
+    height: 100%;
+    padding-top: calc(var(--header-height) + 2rem);
+    width: 15rem;
+    background-color: var(--color-primary);
+    z-index: 1;
+  }
+
+  #nav__menu-toggle:checked + ul {
+    right: 0;
+  }
+
+  .nav__menu > ul > li:not(:last-of-type) {
+    margin-bottom: 1rem;
+  }
+
+  footer {
+    grid-template-columns: auto auto;
+    grid-template-rows: auto auto;
+  }
+
```

Comment:
I think making the footer padding responsive would look better. I tried setting the padding for `footer > *` as `padding-mid` for tablet and it looks decent. 
<img width="850" alt="image" src="https://github.com/JuneOh710/PetPal/assets/82453596/39aec4b9-a6df-45be-adc0-7f95b7d8c512">